src/MICore/CommandFactories/gdb.cs

```diff
@@ -225,5 +225,38 @@ private static string TypeBySize(uint size)
         }
 
         public override bool SupportsDataBreakpoints { get { return true; } }
+
+        public override async Task<TargetArchitecture> GetTargetArchitecture()
+        {
+            string cmd = "show architecture";
```

Comment:
This is the architecture of the target, it will change depending upon the target type.